box-ui/src/components/ts/FloatingBox.tsx

```diff
@@ -5,11 +5,9 @@ import { FloatProps } from '../../utils/Props';
 
 const FloatingBox = ({ children }: FloatProps) => {
```

Comment:
use `const FloatingBox: FunctionComponent = (props) =>` to be consistent with the other components.